src/pfs/pfs.proto

```diff
@@ -320,8 +320,8 @@ message ModifyFileRequest {
 message GetFileRequest {
   File file = 1;
   string URL = 2;
+  int64 offset_bytes = 3;
```

Comment:
"_bytes" might be unnecessary.  There's also a bytes type in protobuf's which is a little confusing.